src/components/Table/Table.test.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import Table from "./Table";
+
+export const columnHeaders = [
+  "First Name",
+  "Last Name",
+  "Nick Name",
+  "Address1",
+  "City",
+  "Zipcode",
+  "State",
+];
+
+export const tableData = [
+  ["Tom", "Nook", "Tanukichi", "Main Street", "New York", "23458", "NY"],
+  ["Isabelle", "-", "Shizue", "Walnut Street", "New York", "23458", "NY"],
+  ["K.K.", "Slider", "Totakeke", "Niper Place", "New York", "98765", "NY"],
+  [
+    "Sonny",
+    "Resetti",
+    "Risetto san",
+    "Village Road",
+    "New York",
+    "09873",
+    "NY",
+  ],
+];
+
+describe("Table Accessibility", () => {
+  it("passes axe accessibility test with the columnHeaders and tableData prop", async () => {
+    const { container } = render(
+      <Table columnHeaders={columnHeaders} tableData={tableData} />
+    );
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("passes axe accessibility test with tableData prop", async () => {
+    const { container } = render(<Table tableData={tableData} />);
+    expect(await axe(container)).toHaveNoViolations();
+  });
+});
+
+describe("Table", () => {
+  const warn = jest.spyOn(console, "warn");
```

Comment:
Move this inside the second test, so inside "tableData prop is empty array" before line 55.